### DISCUSSION TITLE
Looking to help contributing

### DISCUSSION THREAD

**Discussion by 3dAttempt** [freeplane/freeplane#427] [General]
Hello,

I wasn't sure which category to choose so sorry if I chose the wrong one. I would like to help contributing to the project, however I am a beginner and would ask you to give me fairly simple tasks in the beginning, like documentation or tests, so I can improve my skills.
I will be grateful for any answer 🙂

**Comment by dpolivaev** [comment]
There are no "tasks" unless you decide what you want to contribute yourself. You can discuss your ideas with the other community members. If you want to contribute to the documentation, you can write a post in https://github.com/freeplane/wiki/discussions where people write about the next steps related to the documentation.
